ras/smtstate.py

```diff
@@ -12,6 +12,11 @@
 # See LICENSE for more details.
 # Copyright: 2021 IBM
 # Author: Pavithra Prakash <pavrampu@linux.vnet.ibm.com>
+#
+# Modified: 22/11/2022
+# Author: Rohan Deshpande <rohan_d@linux.vnet.ibm.com>
+# Description: Added new test "test_smt_state_switching"
```

Comment:
i feel not require this change as git history contains same

Reply:
Sure, will get it removed.

Reply:
Done